client/blocks/app-banner/style.scss

```diff
@@ -2,52 +2,78 @@
 	position: fixed;
 	bottom: 0;
 	width: 100%;
-	background-color: var( --color-primary-light );
-	color: var( --color-text-inverted );
+	background-color: var( --color-surface );
+	color: var( --color-primary-80 );
+	padding: 20px;
+	margin-bottom: 0;
+	box-shadow: 3px -2px 24px 4px rgba( 0, 0, 0, 0.29 );
+	overflow: hidden;
 	/* Needed for banner to appear in front of notifications panel. */
 	z-index: z-index( 'root', '.app-banner' );
 }
 
-.app-banner__illustration {
-	height: 140px;
+.app-banner__circle {
 	position: absolute;
-	right: 0;
-	z-index: z-index( 'root', '.app-banner__illustration' );
+	border: 2px solid;
+	border-radius: 50%; /* stylelint-disable-line */
+	z-index: z-index( 'root', '.app-banner__background' );
```

Comment:
Should we rename this to `app-banner__circle`?

Reply:
I wasn't sure if the z-index value should match the element name, so I went with something more generic, since it's possible we change the background again in the near future and move away from using the circles. But I guess we can always rename it again if that's the case. Changed in f2b6dfb.